Netkan/CKAN-netkan.csproj

```diff
@@ -159,7 +159,9 @@
     <Compile Include="Validators\VersionStrictValidator.cs" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="../CKAN.schema" />
+    <EmbeddedResource Include="../CKAN.schema">
```

Comment:
Just a very minor comment, the path is a little inconsistent with the majority of the file. Though I note that the before build also uses forward slashes.

I come from a forward slash world, so prefer `../foo` but being C# I'm guessing the common convention is `..\bar`